### ISSUE TITLE
Insert with no primary key fails since update

### ISSUE THREAD

**Issue by Padwah** [schotime/NPoco#270]
I've got a .net application that I've recently update to use NPoco 3.2 from nuget. Unfortunately I've now run in to the following problem. I have the following class:

```
public class UserProviders
{
	public string ProviderName
	{
		get; set;
	}

	public string ProviderKey
	{
		get; set;
	}

	public int UserId
	{
		get; set;
	}
}
```

Which I'm then trying to save using

`Database.Insert(new UserProviders{ ProviderName = "name", ProviderKey = "key", UserId = 1 })`

This code was working fine when using the NPoco v2.10.11 package however since the upgrade now results in the error _Invalid column name 'ID'_ 

I can fix the problem by specifying a composite primary key but was wandering if there's another work around?

**Comment by schotime** [comment]
Does this still happen in 3.3.0-beta?

**Comment by Padwah** [comment]
Just updated to 3.3.0-beta and have the same issue.

**Comment by schotime** [comment]
This has been fixed in 3.3.2.
**Note** this not a supported feature and coincindentally only works in Sql Server when UseOutputClause is off.
